### PR TITLE
Ysimillides patch 1

### PR DESCRIPTION
**Pull request by ysimillides**
removed example folder as discussed. Adding some more operator expressions for geometry + forms.
provided functions to extract arrays from the assembled form( stiffness matrix) and solution.

### CODE REVIEW

**Comment by ChrisRackauckas** [comment]
No new tests?

File: src/jsolve.jl [hunk 1]
```diff
@@ -30,7 +31,27 @@ export File
...
+function get_array(solution::Function)
+    generic_vector = vector(solution)
+    instantiated_vector = Vector(generic_vector)
+    return instantiated_vector[:array]()
+end
+"""
+we return the array from an assembled form
+"""
+function get_array(assembled_form::Matrix)
+    return array(assembled_form)
```
Comment (ChrisRackauckas) [inline src/jsolve.jl]: does this get a sparse matrix?

**Comment by ChrisRackauckas** [comment]
New tests are failing.

**Comment by ysimillides** [comment]
removed the failing test until I can figure out why its not working on Travis

**Comment by ChrisRackauckas** [comment]
>removed the failing test until I can figure out why its not working on Travis

You can instead set it as `@test_broken`.

**Pull request merged**